### ISSUE TITLE
Checkboxes are not clicable (see example)

### ISSUE THREAD

**Issue by lua-rocks** [giuspen/cherrytree#1906]
Here is the example:

[bug.ctb.zip](https://github.com/giuspen/cherrytree/files/7763189/bug.ctb.zip)

The first checkbox is clickable, all the others are not.

**Comment by meijialun** [comment]
you need **3 space** in front of line rather then a tab in the second or third line

**Comment by lua-rocks** [comment]
@meijialun Thank you very much! But is still think this is a wrong behavior. How could I know that there had to be spaces and exact 3? It would be nice if the required indent width and type were matched the ones specified in the program settings.

**Comment by meijialun** [comment]
@lua-rocks just push enter at the end of the line, or press tab if you want retract. You have the wrong case because you copy and paste the content from somewhere else (e.g. microsoft word).
![Snipaste_2022-01-02_21-31-28](https://user-images.githubusercontent.com/5047912/147877403-89fee12b-5956-414b-978c-6db53dc9f863.png)

**Comment by lua-rocks** [comment]
@meijialun no, you misunderstand me. I got it wrong because I set it up wrong. If you open preferences, the first option you can see there is "tab with" and I usually prefer 2 with spaces, but no matter what value you specify - all except 3 with spaces will lead to this issue!

**Comment by giuspen** [comment]
If you press enter for subsequent list items and then click Tab to indent while the cursor is on the list item or Shift+Tab to un-indent it works. Currently this is the only supported mode with 3 spaces for indentation but you don't need to have 3 spaces in the preferences as a tab on a list item is recognised and has a special behaviour

**Comment by lua-rocks** [comment]
@giuspen Yes, I checked, you're right. Basically I'm totally fine with 3 spaces, I just don't think it's good that I must to use only this indent type if I want all the indents in my document to be the same. But of course, it's not very important compared to other issues.

**Comment by Kansai53** [comment]
This is very confusing. Spent awhile trying to figure out why some checkboxes were working and some were not. It seems like if the checkbox is designated as a special character, it should function regardless of if there are exactly 3 spaces in front of it
